plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -227,18 +228,23 @@ def process_main(self):
         available_blobs = set()
         self.total_units = len(available_tags)
```

Comment:
i think total_units should correspond to the units we will work with that in one case would be available_tags and in the other case should be available_tags.intersection(whitelist_tags)